src/ipfs.io/docs/install/index.html

```diff
@@ -10,118 +10,91 @@
 <div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen; line-height: 1.7">
 {% markdown %}
 
-# Install IPFS
+# Install Go IPFS
```

Comment:
maybe in the future we should setup an "ipfs toolchain" that gets installed, similar to git's, which installs:

```
ipfs-update
ipfs (latest)
ipns-pub
```

Other tools we could include:

```
graphmd
bsdash
dhtdash (once it exists)
multihash
multiaddr
```

could be its own dist, called `ipfs-toolchain`